LICENSE.md

```diff
@@ -1,25 +1,127 @@
+deal.II Copyright and License
+=============================
 
 This directory contains the deal.II library.
 
 The deal.II library is copyrighted by the deal.II authors. This term
-refers to the people listed in the file AUTHORS and on the webpage 
-http://www.dealii.org/authors.html.
+refers to the people listed in the file `AUTHORS` and in the
+[authors list](https://www.dealii.org/authors.html) on the webpage.
 
 The deal.II library is free software; you can use it, redistribute it,
-and/or modify it under the terms of the GNU Lesser General Public
-License as published by the Free Software Foundation; either version
-2.1 of the License, or (at your option) any later version.
+and/or modify it under the terms of the <b>GNU Lesser General Public
+License</b> as published by the Free Software Foundation; either <b>version
+2.1</b> of the License, or (at your option) any later version. The full
+text of the GNU Lesser General Public version 2.1 is quoted below.
 
-The full text of the GNU Lesser General Public version 2.1 is quoted
-below.
 
-The subdirectory "bundled/" contains third party software.
-PLEASE NOTE THAT THE SOFTWARE THERE IS COPYRIGHTED BY OTHERS THAN THE
-deal.II AUTHORS, but is included by permission. For details, consult the
-stated licenses there.
 
-----------------------------------------------------------------------
+Contributions
+-------------
 
+As a contributor to this project, you agree that all of your contributions
+be governed by the <b>Developer Certificate of Origin version 1.1</b>.
+The deal.II project does not require copyright assignments for
+contributions. This means that the copyright for code contributions in the
+deal.II project is held by its respective contributors who have each agreed
+to release their contributed code under a compatible open source license
+(LGPL v2.1 for library code). The full
+text of the Developer Certificate of Origin version 1.1 is quoted below.
+
+
+Referencing the library
+-----------------------
+
+In addition to the terms imposed by the LGPL v2.1 or later, we ask for the
+following courtesy:
+
+ -  Every publication presenting numerical results obtained with the help
+    of deal.II should state the name of the library and cite appropriate
+    deal.II references listed at the top of the [publications
+    page](https://www.dealii.org/publications.html).
+
+This is the usual, fair way of giving credit to contributors to a
+scientific result. In addition, it helps us justify our effort in
+developing deal.II as an academic undertaking.
+
+We keep a list of publications using deal.II. Feel free to let us know
+about your publications.
+
+
+Parts of the deal.II distribution under a separate license
+----------------------------------------------------------
+
+The subdirectory `bundled/` contains third party software. <b>Please note
+that the software there is copyrighted by others than the deal.II
```

Comment:
by others than the deal.II authors -> by their respective authors (independent to the deal.II authors)